near/src/runtime.rs

```diff
@@ -588,6 +672,18 @@ impl RuntimeAdapter for NightshadeRuntime {
         }
     }
 
+    fn get_num_validator_blocks(
```

Comment:
this function just got moved around?

Reply:
It's right in the description of the pull request:

> reordering methods to match their order in the RuntimeAdapter trait.